packages/utils/src/types/channelEntry.ts

```diff
@@ -140,6 +155,17 @@ export class ChannelEntry {
     return !!this.closureTime && now.gt(this.closureTime.toBN())
   }
 
+  public getRemainingClosureTime(): BN {
```

Comment:
Please document this functions, specifically the `-1` case.

Reply:
changed !